src/reducers-actions/index.ts

```diff
@@ -0,0 +1,29 @@
+import combineReducers from 'combine-reducers'
+import { reducer as TokenRowReducer, TokenLocalState, TokenRowInitialState as TOKENS } from './tokenRow'
+
+export * from './tokenRow'
+
+export interface GlobalState {
+  TOKENS: TokenLocalState
+}
+
+/**********************************
+ * Initial Global State
+ *
+ * Sets app's initial global state
+ * make sure the name of the state key(s) is/are the same as the reducer key(s) below
+ */
+
+export const INITIAL_STATE: GlobalState = {
+  TOKENS,
```

Comment:
why capitalized?

Reply:
ah i mis-read a previous requested change as "state key should be capitalised" not "it shouldn't be capitalised" - I'll consider the change